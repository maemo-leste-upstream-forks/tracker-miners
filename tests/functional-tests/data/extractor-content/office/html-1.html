<!DOCTYPE html>

<html>
<head>
	
	<meta http-equiv="content-type" content="text/html; charset=utf-8"/>
	<title>Some title</title>
	<meta name="generator" content="LibreOffice 7.4.3.2 (Linux)"/>
	<meta name="created" content="2023-01-13T16:26:55.924680435"/>
	<meta name="changed" content="2023-01-13T16:29:59.467149663"/>
	<meta name="classification" content="Some subject"/>
	<meta name="description" content="Some comment"/>
	<meta name="keywords" content="keyword1, keyword2"/>
	
	<style type="text/css">
		body,div,table,thead,tbody,tfoot,tr,th,td,p { font-family:"Liberation Sans"; font-size:x-small }
		a.comment-indicator:hover + comment { background:#ffd; position:absolute; display:block; border:1px solid black; padding:0.5em;  } 
		a.comment-indicator { background:red; display:inline-block; border:1px solid black; width:0.5em; height:0.5em;  } 
		comment { display:none;  } 
	</style>
	
</head>

<body>
<table cellspacing="0" border="0">
	<colgroup span="3" width="85"></colgroup>
	<tr>
		<td height="17" align="right" sdval="1" sdnum="1033;">1</td>
		<td align="right" sdval="2" sdnum="1033;">2</td>
		<td align="right" sdval="3" sdnum="1033;">3</td>
	</tr>
	<tr>
		<td height="17" align="right" sdval="4" sdnum="1033;">4</td>
		<td align="right" sdval="5" sdnum="1033;">5</td>
		<td align="right" sdval="6" sdnum="1033;">6</td>
	</tr>
</table>
<!-- ************************************************************************** -->
</body>

</html>
