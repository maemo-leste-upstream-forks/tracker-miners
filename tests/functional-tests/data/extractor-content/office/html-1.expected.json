{
    "test": {
        "Filename": "html-1.html",
        "Comment": "HTML document"
    },
    "metadata": {
        "@graph": [
	    {
		"nie:plainTextContent": "1 \n\t\t 2 \n\t\t 3 \n\t \n\t \n\t\t 4 \n\t\t 5 \n\t\t 6",
		"nie:title": "Some title",
		"nie:keyword": [
                    "keyword1",
                    "keyword2"
		],
		"@type": "nfo:HtmlDocument"
	    }
	]
    }
}
